app/assets/stylesheets/admin/customers.css.scss

```diff
@@ -1,3 +1,14 @@
 .tag-with-rules {
   color: black;
 }
+
+table#customers.index {
+
+  tr.customer {
+
+    .guest-label {
+      color: #999;
```

Comment:
Is this color already used somewhere else? besides, it should be stored in a var for better reusability.

Reply:
It looks like none of our admin CSS uses variables for colours... I could add a new colours file and start adding things, but none of it's organised like that for the back end.

Reply:
I think it'd make sense to start doing so hoping to reach 100% one day. At least this will make it much easier to reuse colors and have a sort-of style guide. What do others think?